test/test_extended_models.py

```diff
@@ -205,43 +150,36 @@ def test_old_vs_new_factory(model_fn, dev):
         },
         "quantization": {
             "input_shape": (1, 3, 224, 224),
-            "quantize": True,
         },
         "segmentation": {
             "input_shape": (1, 3, 520, 520),
         },
         "video": {
-            "input_shape": (1, 3, 4, 112, 112),
+            "input_shape": (1, 4, 112, 112, 3),
         },
         "optical_flow": {
             "input_shape": (1, 3, 128, 128),
         },
     }
-    model_name = model_fn.__name__
     module_name = model_fn.__module__.split(".")[-2]
-    kwargs = {"pretrained": True, **defaults[module_name], **TM._model_params.get(model_name, {})}
-    input_shape = kwargs.pop("input_shape")
-    kwargs.pop("num_classes", None)  # ignore this as it's an incompatible speed optimization for pre-trained models
-    x = torch.rand(input_shape).to(device=dev)
-    if module_name == "detection":
-        x = [x]
 
+    kwargs = {**defaults[module_name], **TM._model_params.get(model_name, {})}
+    input_shape = kwargs.pop("input_shape")
```

Comment:
This logic could be simplified but I'm reusing the `TM._model_params` from the other tests. I would prefer to leave this as-is for now and tackle the testing revamping of models as a whole later this half.